packages/ia-components/sandbox/language-select/language-select.js

```diff
@@ -102,37 +101,50 @@ class LanguageSelect extends React.Component {
 
   render() {
     const { selectedLanguage, dropdownOpen } = this.state;
-    const selectedOption = this.languages[selectedLanguage] || this.languages[Object.keys(this.languages)[0]];
+    const selectedOption =
+      this.languages[selectedLanguage] ||
+      this.languages[Object.keys(this.languages)[0]];
 
     return (
-      <div className='language_select'>
+      <div className="language_select">
         <a onClick={this.toggleDropdown}>
-          <LanguageNode name={selectedOption.inLocal} flag={selectedOption.flag} />
+          <LanguageNode
+            name={selectedOption.inLocal}
+            flag={selectedOption.flag}
+          />
         </a>
-        <ul className={dropdownOpen ? 'visible' : null}>
-          {Object.keys(this.languages).map((language) => (
-            <LanguageOption
-               key={`language_${language}`}
-               language={this.languages[language]}
-               value={language}
-               selectedLanguage={selectedLanguage}
-               onClick={this.selectLanguage} />))}
-        </ul>
+        {dropdownOpen && (
+          <ul className={dropdownOpen ? "visible" : null}>
```

Comment:
Now that a short-circuit is used to conditionally render this entire block, the ternary for setting a class on this element is redundant. Remove and set the `className` to always be 'visible'.